source/threads.tex

```diff
@@ -3666,7 +3666,7 @@
 
 \item these actions will not block for the shared state to become ready, except that it
 may block if all of the following are true: the shared state was created by a call to
-\tcode{std::async}, the shared state is not yet ready, and this was the last reference
```

Comment:
We should always qualify `launch::async` to avoid such problems, and then strictly speaking we don't need to qualify the function. But if it helps the reader significantly then we could say "a call to the function template `async`" or something else to disambiguate it.